app/Jobs/Copyright/DeleteCampaignImage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Jobs\Copyright;
+
+use App\Models\Campaign;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+
+class DeleteCampaignImage implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    protected $campaignId;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Request $request)
```

Comment:
Constructor not needed since it's sanitized and de-sanitized directly by the queue